### ISSUE TITLE
File object on Config page goes type "String" instead of "File"

### ISSUE THREAD

**Issue by MasaGon** [ParsePlatform/parse-dashboard#637]
#### Environment Setup
Parse Dashboard 1.0.22 on Windows 10, Node.js v6.9.2
Parse Server v2.3.2 on Amazon Linux, EC2, wrapped with Elastic Beanstalk

#### Steps to reproduce
1. Go config page on Parse Dashboard
2. Click "Create a parameter"
3. Parameter Name: "test", Type: "File" then "Upload a file" and click "Create parameter"
4. Reload page and you should see type of "test" parameter go "String" instead of "File"

- [ x] You're running version >=1.0.21 of Parse Dashboard.
- [ x] You're running version >=2.2.24 of Parse Server.
- [ x] You've searched through [existing issues](https://github.com/ParsePlatform/parse-dashboard/issues?utf8=%E2%9C%93&q=). Chances are that your issue has been reported or resolved before.

**Comment by mross22** [comment]
I saw this issue as well.  After doing some investigating, I think this is an issue in parse-server itself and not parse-dashboard.

In parse server, when transforming a 'File' object to be stored in the database it looks like it is converted to only the value of the 'name' field which appears to be the root cause of the issue you are reporting.  (see FileCoder.jsonToDatabase):
https://github.com/ParsePlatform/parse-server/blob/4cb6e7d209903990c02e2e3f1a9bcd4aa03983aa/src/Adapters/Storage/Mongo/MongoTransform.js

If you look in your underlying mongoDB, you can see that when you create a config value of type 'File' in the dashboard, it correctly creates the file, it is just referenced improperly in the '_GlobalConfig' collection.

For example, if you create a 'File config value named 'test', your '_GlobalConfig' in mongo will have a field something like:
"test": "41c9ddbf510d26cabad1cec6f2244dc1_test.json"

If you manually update this in the database to be of the form:

"test" : {
_type: "__File",
name:"41c9ddbf510d26cabad1cec6f2244dc1_test.json",
url: "{parse server url}/parse/files/{appid}/{file name from above}"
}

and then refresh parse dashboard in the browser, you will now see that everything works properly.

**In summary,** it looks like parse-server is failing to record the config value as type 'File' with a 'name' and 'url' field so even though the File itself is stored, the dashboard doesn't know where to find it.

I will file an issue on Parse-Server
}

**Comment by mross22** [comment]
Here is the tracking parse-server issue:
https://github.com/ParsePlatform/parse-server/issues/3457

**Comment by flovilmart** [comment]
Closing as tracked on parse-server

**Comment by flovilmart** [comment]
Thanks @mross22 do you want to take on the fix?

**Comment by mross22** [comment]
sure, I'll follow up on the other thread and send out a PR

**Comment by natanrolnik** [comment]
@mross22 @flovilmart this was actually a bug by itself, not related to ParsePlatform/parse-server#3457. 
Now that the creation of a File and setting it to config works, the Dashboard still displays it as a stringified version of the File object.

I've fixed it and opened a PR #666

**Comment by natanrolnik** [comment]
Fixed in the latest release

**Comment by Abderezai** [comment]
broken again

**Comment by flovilmart** [comment]
@Abderezai that's not what I would say a proper issue report.
